README.md

```diff
@@ -41,6 +41,16 @@ All requirements are installed as part of `pip install` or if you're planning on
 * schema
 * yara-python
 
+**Note: Windows users will need to have [Microsoft Visual C++ 14.0 Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools) installed in order to compile yara-python.**
+
+**Note: Linux users will need to have the python developer package installed in order to compile yara-python. If there are compile errors make sure to be on the latest gcc compiler**
```

Comment:
Consider changing second sentence to "If you receive compile errors, make sure you are on the latest..."